src/dataExplorer/components/SaveAsCellForm.tsx

```diff
@@ -234,9 +261,10 @@ const mdtp = {
   onGetDashboards: getDashboards,
   onCreateCellWithView: createCellWithView,
   onCreateDashboardWithView: createDashboardWithView,
+  setTimeMachine: setActiveTimeMachine,
```

Comment:
any reason to alias this? there are no naming conflicts, and i can't think of any good reasons to remove `active` and the implications of having that in the function name.

Reply:
yes, its annoying to me to have the imported method and the method in the component have the same name.

Reply:
@ShmuelLotman what about this having the same name is annoying to you?

it's annoying to me to lose track of a reference to a function when i'm globally searching a codebase.

if i'm trying to take stock of all the places where the method `setActiveTimeMachine` is called, i now have to account for this renaming that doesn't exist for any practical reason. there's no naming collision in this file and so there's no obvious practical reason to do it. it doesn't make the code more [self documenting](https://github.com/influxdata/ui/blob/master/src/billing/thunks/index.ts#L23-L25) like some aliases in our codebase might be argued to do.

Reply:
you dont lose track to the reference just because the name changes there because you still have it imported and its still being used in the file, so the file will still have reference to it. Feel somewhat like shadowing (even though its not) and it doesn't feel like clean code to have a mapped function have the same name. I dont think you lose anything by renaming it, and i think you gain the clarity of knowing you are calling the mapped function instead of looking like you are referencing the imported one without mapping.

Reply:
That being said, thats just my personal opinion and i changed it in the latest push to reflect your request. I guess if you have this.props its easier to see, but this should ideally eventually make it into a functional component and there, particularly with destructuring and/or using useDispatch, this will look different anyways.